hikari/guilds.py

```diff
@@ -515,6 +517,16 @@ def colour(self) -> colours.Colour:
         return self.color
 
 
+class IntegrationType(str, enums.Enum):
+    """The integration type."""
+
+    TWITCH = "twitch"
+
+    YOUTUBE = "youtube"
```

Comment:
just a slight nitpick about the lack of docs for the individual attributes, not sure what this is defaulting to these days but even a basic description would probably look better than the default